docs/source/en/perf_infer_gpu_one.md

```diff
@@ -36,8 +38,79 @@ model = model.reverse_bettertransformer()
 model.save_pretrained("saved_model")
 ```
 
+### `BetterTransformer` API and Flash attention for decoder models
+
 As of PyTorch 2.0, the attention fastpath is supported for both encoders and decoders. The list of supported architectures can be found [here](https://huggingface.co/docs/optimum/bettertransformer/overview#supported-models).
 
+For decoder-based models (e.g. GPT, T5, Llama, etc.), the `BetterTransformer` API will convert all attention operations to use the [`torch.nn.functional.scaled_dot_product_attention` method](https://pytorch.org/docs/master/generated/torch.nn.functional.scaled_dot_product_attention) (SDPA), that is available only from PyTorch 2.0 and onwards. 
```

Comment:
Same comments for the rest of this section as in `perf_infer_gpu_many.md` (you can probably copy the changes over) :)